internal/api/controllers/private/runsCreateActions.go

```diff
@@ -122,27 +122,27 @@ func validateSatelliteFields(runInput RunInputV2) error {
 	return nil
 }
 
-func runCreateError(code int) *RunCreated {
+func runCreateError(code int, message string) *RunCreated {
 	return &RunCreated{
-		Code: code,
-		// TODO report error
+		Code: 	code,
+		Message: message,
 	}
 }
 
 func handleRunCreateError(err error) *RunCreated {
 	if _, ok := err.(*dispatch.RecipientNotFoundError); ok {
-		return runCreateError(http.StatusNotFound)
+		return runCreateError(http.StatusNotFound,"Receipient not found")
 	}
 
 	if _, ok := err.(*tenantid.TenantNotFoundError); ok {
-		return runCreateError(http.StatusNotFound)
+		return runCreateError(http.StatusNotFound,"Tenant not found")
 	}
 
 	if _, ok := err.(*utils.BlocklistedOrgIdError); ok {
-		return runCreateError(http.StatusBadRequest)
+		return runCreateError(http.StatusBadRequest,"Block listed org")
 	}
 
-	return runCreateError(http.StatusInternalServerError)
+	return runCreateError(http.StatusInternalServerError,"Host Disconnected")
```

Comment:
I'm wondering if we should make this one more generic.  :thinking:   Maybe something like "An unexpected error happened during processing"?

Reply:
we can also do "unexpected Error during processing"? @dehort also are the other three messages good? I didnt know if we needed to change those or not

Reply:
Yep, I think the other messages look good.